server/application/application.proto

```diff
@@ -19,6 +19,12 @@ message ApplicationQuery {
 	optional string name = 1;
 }
 
+// ApplicationEventsQuery is a query for application resource events
+message ApplicationResourceEventsQuery {
+	required string appName = 1;
+	required string resName = 2;
```

Comment:
I've realized that resource name is not enough to uniquely identify an object. Please add resource kind and use it in `ListResourceEvents` method.